newsfragments/1444.feature

```diff
@@ -0,0 +1 @@
+dials.merge: Add best_unit_cell option. Allow this to be set from dials.scale also.
```

Comment:
> Allow this to be set from dials.scale also.

Does this mean the user can control it from dials.scale, or do you just mean that dials.scale will now use the best unit cell when outputting the merged mtz file?